lib/facility.rb

```diff
@@ -1,14 +1,21 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :vehicle, :registered_vehicle
```

Comment:
what is this attr_reader for :vehicle for? I don't see anywhere where you need to call `.vehicle` on a Facility object, so you can remove that.